.travis.yml

```diff
@@ -4,17 +4,16 @@
 language: objective-c
 
 env:
-  matrix:
-    
-    - CONDA_NPY=110  CONDA_PY=27
```

Comment:
I think this is perfectly reasonable. This feedstock no longer has any dependence upon Python - that is all done in the python-ecmwf_grib-feedstock.